docs/src/integrations/activity/_index.md

```diff
@@ -107,6 +107,10 @@ As a general rule, it is better to have an activity script only execute on the s
 
 Activity scripts have a series of APIs available to them to facilitate building out custom functionality.
 
+### `underscore.js`
+
+Underscore.js 1.9.2 is available out-of-the-box to make writing Activity scripts more pleasant.  See [Underscore's documentation](https://cdn.rawgit.com/jashkenas/underscore/1.9.2/index.html) for available functions and utilities.
```

Comment:
```suggestion
Underscore.js is available out-of-the-box to make writing Activity scripts more pleasant.  See [Underscore's documentation](https://cdn.rawgit.com/jashkenas/underscore/1.9.2/index.html) for available functions and utilities.
```

Reply:
1.9.2 is not the latest version, and I don't expect us to be updating it as frequently as they do.  Hence the specific version callout.